data/sql/updates/pending_db_world/rev_XXXXXXXXXXXX.sql

```diff
@@ -0,0 +1,9 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('XXXXXXXXXXXX');
+
+DELETE FROM `gameobject_loot_template` WHERE `entry` IN (11104); -- Secret Safe
+INSERT INTO `gameobject_loot_template` (`entry`, `item`, `chance`, `questrequired`,  `groupid`, `mincount`, `maxcount`) VALUES
+(11104, 22205, 25, 0, 1, 1, 1),
+(11104, 22255, 25, 0, 1, 1, 1),
+(11104, 22256, 25, 0, 1, 1, 1),
+(11104, 22254, 25, 0, 1, 1, 1),
+(11104, 11309, 100, 1, 1, 1, 1);
```

Comment:
all the items are in groupid = 1, so it means only one item will be chosen

and since you put chance 100 to item 11309, it will always be this item, so there is soemthing wrong too here (otherwise thank you already)

Reply:
Well, only 1 should be selected, and the one that has 100 also has QuestRequired = 1 so it should not interfere with the other's

Reply:
The only case where there should be two drops from what I understand is if the player has the quest required for item 11309

Reply:
my bad loool, sorry I had not verified the loot from wowpedia/wowhead before talking

Reply:
:P

Reply:
Ok so I checked the loot from wowhead, the chances are a bit different :D 26, 24, 24, 23
But I guess it doesn't matter

HOWEVER i thought about this thing again, and the quest item must be in groupid=0 and not in groupid = 1 else it means when you have the quest, it will not drop any blue item, only the quest item

In top of that, I verified how it is for another boss and it's done this way (blue items in groupd id 1 and quest item in groupd id 0)

Reply:
WoW Head sucks when it comes to chances because their "addon" that collects data some users use on private servers which make the chances very off in some cases.

Reply:
ok and we can see that it's probably intended to be 25 25 25 25 (and it makes sense)

BUT THEN, you could as well put chance = 0, which means equal between all (100/4 = 25 :P)